page-optimize.php

```diff
@@ -52,7 +52,19 @@ function page_optimize_should_concat_css() {
 		return $_GET['concat-css'] !== '0';
 	}
 
-	return !! get_option( 'page_optimize-css' );
+	return !! get_option( 'page_optimize-css', page_optimize_css_default() );
+}
+
+function page_optimize_js_default() {
```

Comment:
I like these functions because we can easily expose a filter to change the defaults.